ballerina/tests/resources/generated-sources/tool_test_external_imports/parent_pb.bal

```diff
@@ -0,0 +1,239 @@
+import ballerina/grpc;
```

Comment:
licence block is missing here also.

Reply:
These are test resource files that are used to asset the content of the bal files generated from the gRPC tool. Therefore they don't have license headers.